### PR TITLE
Fix shellscript in '📦 Check Pods Manifest.lock'

### PR DESCRIPTION
**Pull request by webmaster128**
This fixes two minor issues in the script as described in the commit messages.

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
        
            1 Error
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! \nYou can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
    
    
  </tbody>
</table>
  

  

  



Here's an example of your CHANGELOG entry:

```markdown
* Fix shellscript in '📦 Check Pods Manifest.lock' 
  [webmaster128](https://github.com/webmaster128)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.




<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by segiddins** [comment]
👍 with a CHANGELOG, we'll also have to rebuild the integration specs

**Comment by webmaster128** [comment]
Thanks for feedback. I am a novice in all this CocoaPods Ruby magic. Is there anything else apart from the CHANGELOG entry, I can do?

File: lib/cocoapods/installer/user_project_integrator/target_integrator.rb [hunk 1]
```diff
@@ -157,10 +157,9 @@ def add_check_manifest_lock_script_phase
             native_target.build_phases.unshift(phase).uniq! unless native_target.build_phases.first == phase
             phase.shell_script = <<-SH.strip_heredoc
               diff "${PODS_ROOT}/../Podfile.lock" "${PODS_ROOT}/Manifest.lock" > /dev/null
-              if [[ $? != 0 ]] ; then
-                  cat << EOM
-              error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.
-              EOM
+              if [ $? != 0 ] ; then
+                  # print error to STDERR
+                  echo "error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation." >&2
```
Comment (neonichu) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: Any reason why this was changed to `echo`?

**Comment by webmaster128** [comment]
Check

File: lib/cocoapods/installer/user_project_integrator/target_integrator.rb (hunk 1 above)
Comment (orta) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: ^ thanks 👯

**Comment by segiddins** [comment]
👍🏻

**Pull request merged**